project-base/storefront/components/Blocks/UserNavigation/UserNavigation.tsx

```diff
@@ -52,7 +53,10 @@ export const UserNavigation: FC = () => {
                         href={menuItem.link}
                         type={menuItem.type}
                     >
-                        {menuItem.text}
+                        <div>
```

Comment:
is this div necessary?

Reply:
Yep thats because of flex. Maybe there is better way to correctly align whole row, so you could tell me how to do it

Reply:
pushed fix, check it out :)